### DISCUSSION TITLE
strange cncl encountered in mmengine/dist/utils.py

### DISCUSSION THREAD

**Discussion by syboomsy** [open-mmlab/mmengine#1104] [Q&A]
![image](https://user-images.githubusercontent.com/57123221/233972406-c6984fbe-8049-4855-9235-b46f47a6fdeb.png)
Please tell me whether the default backend cncl is the result of wrongly typing 'nccl or it is just another derived backend's nickname?

**Comment by zhouzaida** [comment]
This is another device and its communication backend is `'nccl'`.
